app/views/topics/propose_archive.html.erb

```diff
@@ -1,10 +1,24 @@
 <%= tag_header @archival.tag, 'Archive topic' %>
```

Comment:
Archive is the correct word, the description needs to be updated. We're not really deleting them because they're still accessible from the publisher UI.